gradle/lib/dependabot/gradle/file_fetcher.rb

```diff
@@ -29,6 +29,7 @@ def self.required_files_message
       def fetch_files
         fetched_files = []
         fetched_files << buildfile if buildfile
+        fetched_files << settings_file if settings_file
```

Comment:
This is already pulled in via the `subproject_buildfiles` method below I think?

```suggestion
```

Reply:
Yes, the settings files are fetched by `subproject_buildfiles` but in my opinion, without adding to fetched_files, the files will not be parsed. I'll check if adding to fetched_files is required or not.

Reply:
Without this line, parsing settings files will not be performed so it looks this line is required.